lighting/raymarch/glass.glsl

```diff
@@ -0,0 +1,200 @@
+#include "../envMap.glsl"
+#include "../fresnel.glsl"
+#include "../specular.glsl"
+#include "../reflection.glsl"
+
+/*
+original_author:  The Art Of Code
```

Comment:
Martijn Steinrucken ?

Reply:
Yes, the guy! I looked up on his [Twitter](https://twitter.com/The_ArtOfCode), his name is `Martijn Steinrucken`